### DISCUSSION TITLE
How to run the UI/dashboard?

### DISCUSSION THREAD

**Discussion by ZedZipDev** [datalust/seq-tickets#1401] [Help]
I have Windows 10, I run the docker:
>docker run --name seq -d --restart unless-stopped -e ACCEPT_EULA=Y -p 5341:80 datalust/seq:latest

(There is : https://hub.docker.com/r/datalust/seq)

Now I try to run the UI 
htp://localhost:5341

But  receive
"This page is not working"

What do I wrong? How to run the UI?

**Comment by nblumhardt** [comment]
Duplicate of: https://github.com/datalust/seq-tickets/issues/1402
